docs/component_conventions.md

```diff
@@ -188,6 +226,16 @@ Follow the [GOV.UK Frontend JS conventions](https://github.com/alphagov/govuk-fr
 
 Scripts should use the [module pattern provided by govuk_frontend_toolkit](https://github.com/alphagov/govuk_frontend_toolkit/blob/master/docs/javascript.md#modules) and be linted using [StandardJS](https://standardjs.com/).
 
+Most components should include tracking (see the [select component](https://govuk-publishing-components.herokuapp.com/component-guide/select/with_tracking) for [example code](https://github.com/alphagov/govuk_publishing_components/blob/master/app/assets/javascripts/govuk_publishing_components/components/select.js)).
```

Comment:
Tracking can have a negative impact on users (privacy, performance).

We already have a problem with things being tracked without a real reason for doing so, I think you should reconsider making tracking a default in components.

Could we make this clear that tracking is only added to components that can be tied to something a team needs to know, and have a process to review and remove that?